helpers/build_bootimg_packages.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# build_bootimg_packages.sh - Build and/or install extra packages
+# needed for droid-hal-img-boot
+
+if [[ ! -d rpm/dhd ]]; then
+    echo $0: 'launch this script from the $ANDROID_ROOT directory'
+    exit 1
+fi
+# utilities
+. ./rpm/dhd/helpers/util.sh
+
+sb2 -t $VENDOR-$DEVICE-$PORT_ARCH -R -msdk-install zypper ref -f
+
+sb2 -t $VENDOR-$DEVICE-$PORT_ARCH -R -m sdk-install zypper -n install -f \
+    e2fsprogs openssh-clients openssh-server
```

Comment:
Thank you for your contribution.

There is no need to refresh and install, as these packages are already available and will be pulled in automatically by the builds below. I'll push the fix to your branch and merge this PR.